### PR TITLE
fix(curriculum): specify id attribute in the instructions

### PR DESCRIPTION
**Pull request by chinmaykunkikar**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/jquery/remove-an-element-using-jquery.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Now let's remove an HTML element from your page using jQuery.
 
 jQuery has a function called `.remove()` that will remove an HTML element entirely
 
-Remove element `target4` from the page by using the `.remove()` function.
+Remove element with id `target4` from the page by using the `.remove()` function.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/03-front-end-libraries/jquery/remove-an-element-using-jquery.md:15]:
```suggestion
Remove the `#target4` element from the page by using the `.remove()` function.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for opening this PR.

If we use the word `id` we would need to include it within backticks for translation. This way, we do not need to do that.

File: curriculum/challenges/english/03-front-end-libraries/jquery/remove-an-element-using-jquery.md (hunk 1 above)
Comment (nhcarrigan) [reply curriculum/challenges/english/03-front-end-libraries/jquery/remove-an-element-using-jquery.md:15]: @chinmaykunkikar if you agree with Shaun's suggestion above, could you go ahead and commit it?

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Comment by ShaunSHamilton** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**